frontend/src/layout/navigation/SideBar/SideBar.tsx

```diff
@@ -224,6 +224,7 @@ function Pages(): JSX.Element {
             {(hasAvailableFeature(AvailableFeature.EXPERIMENTATION) ||
                 !preflight?.instance_preferences?.disable_paid_fs) && (
                 <PageButton
+                    title="Experimentation"
```

Comment:
Instead of adding this manually to this `PageButton`, better to update `sceneConfigurations` – then the breadcrumb will be correct too!

Reply:
makes a lot of sense, just did!